src/web/app/page.component.html

```diff
@@ -67,9 +67,14 @@ <h1 *ngIf="pageTitle && isValidUser">{{ pageTitle }}</h1>
   <footer>
     <div class="container footer text-nowrap">
       <div class="row">
-        <div class="col-xs-12 col-sm-4 offset-sm-8 col-md-3 offset-md-9">
-          Hosted on <a class="footer" href="https://cloud.google.com/appengine/" target="_blank" rel="noopener noreferrer">Google App Engine</a>
+        <div class="col-sm-4 col-md-4 text-left">
+          <i class="fas fa-bolt"></i><a routerLink="/web/front/home"> TEAMMATES</a> v{{ version }}
```

Comment:
Minor issue, but let's upcase the `v` before version to make it similar with V6.

Reply:
Fixed!